hash/sega_beena_cart.xml

```diff
@@ -6,27 +6,172 @@ license:CC0
+				<rom name="0 - 0002.png" size="0x1fde709" crc="72778f05" sha1="998f3c6bbd5bdabf1150fe5dcfa5eee0dfc85083"/>
+				<rom name="0 - 0003.png" size="0x1fbfd28" crc="e4f1a9b4" sha1="e88da1ee32615d263b616fc495f5f271892b7d82"/>
+				<rom name="0 - 0004.png" size="0x1ef0e79" crc="1945ee85" sha1="23f2db46075b1d5be48a0df680235b8dc39b62d2"/>
+				<rom name="0 - 0005.png" size="0x1fbfd2b" crc="4fba7dfa" sha1="bf86eb4bb3f278f425c666b13bf7dd8ecb8ca557"/>
+				<rom name="0 - 0006.png" size="0x1be6425" crc="5e439b1e" sha1="dc2e26a93644bd02b6fc4cb803509457863af333"/>
+				<rom name="0 - 0007.png" size="0x1cd7f10" crc="140f5dca" sha1="76e5ebf5fa61fc7c0962cb222d9836df1be62d30"/>
+				<rom name="0 - 0008.png" size="0x203de88" crc="f8b7cca2" sha1="62be480fe570c4a3724693379fe2917af21c428f"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="dtanoshib" supported="no">
+		<description>Disney Tanoshii O-Benkyou Series: Mic de Kantan ABC</description>
+		<year>2005</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ディズニー たのしいおべんきょうシリーズ ① マイクでかんたんABC♪"/>
+		<part name="cart" interface="sega_beena_cart">
+			<dataarea name="rom" size="0x800000" width="32" endianness="big">
+				<rom loadflag="load16_word_swap" name="S-100005-1000 - Disney Tanoshii o-Benkyou Series-Mic de Kantan ABC (Japan).bin" size="0x800000" crc="e95e91a0" sha1="ea3dcaca3c1fcc31dbc58674238c7c32d12539ad"/>
+			</dataarea>
+			<dataarea name="pages" size="0x2500000"> <!-- book pages -->
+				<rom name="0 - 0001.png" size="0x1eeef5b" crc="e3e3bda5" sha1="01588ca8952b15cf96385c16d4f84ffcc84577ef"/>
+				<rom name="0 - 0002.png" size="0x1e7b04a" crc="7587cf18" sha1="e76698fe50b2f35900bdab736f4099540155f3fd"/>
+				<rom name="0 - 0003.png" size="0x2198fe7" crc="bafc211e" sha1="a8a177841bb634f973f0fa1267648cc4d455a812"/>
+				<rom name="0 - 0004.png" size="0x2100ccd" crc="f0a53d77" sha1="33beb3c2a89dcd35c39092bc9fab5f4608993794"/>
+				<rom name="0 - 0005.png" size="0x2031a2d" crc="f1a4ae68" sha1="0143c4ff729e80a5c1c49e41484e6cc7c33ed72e"/>
+				<rom name="0 - 0006.png" size="0x1e698ee" crc="dd6cb40e" sha1="17e25c3ad90b3ee7f60208cca67b611bfe4dd8a9"/>
+				<rom name="0 - 0007.png" size="0x1f1492e" crc="06dc678d" sha1="2c9009604acfa517a407951e7a5bff95718ab2ab"/>
+				<rom name="0 - 0008.png" size="0x1d92fc9" crc="d3bfc642" sha1="4bcecd682712bd06f6d9901af3ede91efbb35682"/>
+				<rom name="0 - 0009.png" size="0x1dc3333" crc="0a59e863" sha1="4ccce518e2b53494c1f68bbc742b398cd6bfa202"/>
+				<rom name="0 - 0010.png" size="0x1f05eea" crc="541f1068" sha1="22f50cd233c9059129cb90bf4a7d62b9fa04ff71"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="dtanoshimt" supported="no">
+		<description>Disney Tanoshii Oekaki: O-Mise-ya-san o Tsukutchaou!</description>
+		<year>2007</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ディズニーたのしいおえかき おみせやさんをつくっちゃおう!"/>
```

Comment:
Please use fullwidth punctuation in Japanese unless there’s clear evidence that they used a halfwidth form (in this case, the exclamation mark).

Reply:
Replaced all halfwidth with regular chars and removed spaces!